ipatests/pytest_ipa/integration/tasks.py

```diff
@@ -2053,6 +2055,8 @@ def uninstall_packages(host, pkgs):
     # Only supports RHEL 8+ and Fedora for now
     if platform in ('rhel', 'fedora'):
         install_cmd = ['/usr/bin/dnf', 'remove', '-y']
+    elif platform in ('ubuntu'):
+        install_cmd = ['apt-get']
```

Comment:
This line should be `install_cmd = ['apt-get', 'remove', '-y']`

Reply:
done